### DISCUSSION TITLE
Standalone screen search app

### DISCUSSION THREAD

**Discussion by g-bortoletto** [adirh3/Fluent-Search#677] [General]
I would really like a standalone program with just the search screen (default Ctrl+M) functionality. Are there any replacements or plans to make something like this?

**Comment by adirh3** [comment]
@g-bortoletto I plan to also publish screen search separately, only need to think of a name for it :)

**Reply by g-bortoletto** [reply]
I'd call it Jumpuh!, but idk. Just please release it 🙏

**Reply by adirh3** [reply]
Nice idea lol, been working on it, but it may take time
